elements/pfe-styles/src/pfe-base.scss

```diff
@@ -84,8 +84,8 @@ th {
 body {
   @extend %pfe-content-base;
   /* This variable is intentionally left empty for light DOM within components */
-  font-size: var( --pf-c--FontSize, pfe-var(content--FontSize));
-  color: pfe-var(text);
+  font-size: var(--pf-c--FontSize, pfe-var(content--FontSize));
+  color: var(--pfe-broadcasted--text, #{pfe-var(text)});
```

Comment:
@starryeyez024 I was doing a little work on the baseline images and realized we didn't include the broadcast variable in the base styles.  Do you think this is an okay update?  I can pull this out into a separate PR if that's easier.

Reply:
In theory, we only need the broadcast variable if the context is reversed, which only happens within container components. So I thought we only needed components like band, card, accordion, etc. to reset the color property with broadcast vars?

Are you thinking that if we added this, we could reduce the duplication of setting the broadcast vars in container components completely? Also yes let's make this a separate PR if possible so we can separate concerns, since this feels like an important change. :)

Reply:
The `color` property cascades so my thought was, text inside a context won't necessarily have `color` styles due to potentially not being a direct child of the slot, but if it's at least defined in base, we have a chance at influencing the text color.

Reply:
Oh and we would still need to update the broadcast vars though in the component, this would just adds the empty hooks to the stylesheet.

Reply:
https://github.com/patternfly/patternfly-elements/pull/1530